subprojects/language-native/src/integTest/groovy/org/gradle/language/swift/AbstractSwiftIntegrationTest.groovy

```diff
@@ -48,6 +48,23 @@ abstract class AbstractSwiftIntegrationTest extends AbstractNativeLanguageCompon
         failure.assertThatCause(Matchers.containsText("Swift compiler failed while compiling swift file(s)"))
     }
 
+    def "binaries have the right Swift version"() {
```

Comment:
should apply also to XCTest suites, just like the check for target platform and tool chain. Adding it here will apply the test only to the main component rather than all Swift components.